helm/chart-operator/templates/priorityclass.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.e2e }}
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: giantswarm-critical
+  labels:
+    {{- include "chart-operator.labels" . | nindent 4 }}
+value: 1000000000
+globalDefault: false
+description: "This priority class is used by giantswarm kubernetes components."
+{{- end }}
```

Comment:
More prep for helm 3. As we don't install tiller we need to create the priority class in the chart for integration tests.

Reply:
For my interest: why we don't need this with tiller?

Reply:
The priority class is one of several created by EnsureTillerInstalled. So when it gets removed it doesn't get created.

This is only for the integration tests because in production the priority class is created via ignition.